### PR TITLE
Added exchanges for ME, capacities for ME and RS

### PR DESCRIPTION
**Pull request by Pantkowsky**
Fix for #138 

Implementations:

- Added ME capacities
- Added RS capacities
- Added exchanges between ME and AL, BA, RS

untitled

untitled2

### CODE REVIEW

**Comment by corradio** [comment]
@ThierryOllivero please don't merge master into a PR. Only let the author do it, as it might have unexpected consequences.

**Comment by ThierryOllivero** [comment]
@corradio : you might be referring to the Albania - Serbia arrow ? Which is a bit weird, since Albania has a border with Kosovo but not Serbia (and we have to keep in mind that Serbia still consider Kosovo to be part of Serbia...).

**Comment by Pantkowsky** [comment]
@corradio If it is then I will change it right away. I honestly thought that the direction is automatically set by code, which determines who imports/exports what at the current time, and the only necessary thing was to set the angle ! If you know for sure that it is wrong and should be set in the opposite direction then please confirm and I will fix it immidiately !

**Comment by corradio** [comment]
@thierryollivero I'm referring to al/me. I know the rs/al sounds weird, but entsoe has it that way. Would be nice to check though.
@apant the direction is indeed wring. You can see that me imports from AL on your screenshot. Yet the arrows shows a different direction. You can also see this in the color of the arrow, which is black. The color is always the color of the origin country. You should therefore add/remove 180 to the arrow angle.

**Comment by ThierryOllivero** [comment]
According to OST, the TSO of Albania:

In the establishment of the above lines are incorporated as well the interconnection lines with the neighboring countries as follows:

Interconnection Line of 220 kV, Fierza (Albania) – Prizren (Kosovo)

Interconnection Line of 220 kV, Koplik (Albania) – Podgorica (Monte Negro)

Interconnection Line of 400 kV, Zemblak (Albania) – Kardia (Greece)

Interconnection Line of 400 kV, Tirana (Albania) – Podgorica (Monte Negro)


So, the arrow should go from Albania to Kosovo on the map, I'll create a specific issue on that and I'll correct it.

**Comment by corradio** [comment]
It's starting to be a mess with all the arrows :(
But looks good to me!

image

**Comment by corradio** [comment]
@brunolajoie can you PLEASE check capacities? It takes 2min.

File: web/app/configs/capacities.json [hunk 1]
```diff
@@ -183,6 +183,12 @@
             "wind": 55
         }
     },
+    "ME": {
+        "capacity": {
+            "coal": 210,
+            "hydro": 342
```
Comment (brunolajoie) [inline web/app/configs/capacities.json]: Suggest update to ENTSOE & IRENA data: Hydro: 652

File: web/app/configs/capacities.json [hunk 2]
```diff
@@ -239,6 +245,13 @@
             "wind": 2938
         }
     },
+    "RS": {
+        "capacity": {
+            "coal": 5644,
+            "hydro": 3006,
```
Comment (brunolajoie) [inline web/app/configs/capacities.json]: Suggest update to ENTSOE 2016 data: Hydro: 2917MW

File: web/app/configs/capacities.json [hunk 3]
```diff
@@ -239,6 +245,13 @@
             "wind": 2938
         }
     },
+    "RS": {
+        "capacity": {
+            "coal": 5644,
```
Comment (brunolajoie) [inline web/app/configs/capacities.json]: Suggest update to ENTSOE 2016 data: Coal: 5666MW.

File: web/app/configs/capacities.json [hunk 4]
```diff
@@ -239,6 +245,13 @@
...
+            "hydro": 3006,
+            "unknown": 428
```
Comment (brunolajoie) [inline web/app/configs/capacities.json]: "Unknown" is actually the "Panonske TE-TO power plant", which is a modern (2015) mixed Gas & Oil powered Combined Heat & Power plant, generating 424 MWe and 505 MWth in paralell. Most of its primary fuel is Gas. I suggest to consider it as Gas-electricity, with an IPCC median value for emission intensity of 490gCO2eq/kWhe. We will not account for the "free" heat but for the moment, heat is out of the scope of electricity map.

Comment (Pantkowsky) [inline web/app/configs/capacities.json]: I will change "unknown" to "gas" then. Just so I understand you correctly @brunolajoie - you suggested to change the value of unknown/gas to 490, or 424/505 ?

**Comment by corradio** [comment]
thanks for the changes @Pantkowsky ! @brunolajoie do you validate?

File: web/app/configs/capacities.json (hunk 4 above)
Comment (brunolajoie) [inline web/app/configs/capacities.json]: Perfect, we use IPCC median values, so 490 will be indeed the emission intensity used for gas. Thanks!

**Pull request merged**